GcmEndpoints/README.md

```diff
@@ -7,7 +7,9 @@ This backend template employs [Google Cloud Endpoints](https://developers.google
 
 # 1. Adding a backend in Android Studio
 
-To add the backend to your existing Android app from this backend template, open Android Studio ([installation instructions](https://developer.android.com/sdk/installing/studio.html)) and navigate to "File &rarr; New Module..." or right-click on your project and choose "New &rarr; Module".
+Start with an existing Android app. You can use the [GCM sample Quickstart app](https://github.com/googlesamples/google-services/tree/master/android/gcm) as a starting point.
+
```

Comment:
I wonder if it's a good idea to include messaging that tells the user that they HAVE to first make sure their app can accept gcm messages (if they're not using the quickstart guide) or these instructions wont make sense.

"This guide builds on top of GCM sample quickstart app, you can also work with an existing app following their guide to add gcm client code to your application" or something <-- not even really sure they have a guide to modify an existing app.